setup.py

```diff
@@ -23,6 +23,16 @@
     'apache-beam[gcp]>=2.2',
     'intervaltree>=2.1.0,<2.2.0',
     'pyvcf<0.7.0',
+    'google-api-python-client>=1.6',
+    # TODO(bashir2): It seems that setuptools cannot find the intersection of
+    # dependency requirements. For example the following dependencies are
+    # listed only because installing google-api-python-client brings versions
+    # of these packages that are newer than versions required in apache-beam
+    # or its dependencies. I think there should be a better of doing this that
+    # I am not aware of!
```

Comment:
@arostamianfar please comment if you know a better way for doing this; thanks.

Reply:
I don't think these are needed. I was able to run the tests on a clean VM after fixing some other issues (although, table deletion still fails at the end of the run). Sent https://github.com/googlegenomics/gcp-variant-transforms/pull/70 . Added some more TODOs for you as well :)

Reply:
Hmm, then I might be missing something here, because I also tried in a clean VM and got similar errors as before (in my clean git directory on my local machine). Here are the steps:

- Getting a new VM on GCP and SSH into it.
- $ sudo apt-get install git
- $ git clone https://github.com/googlegenomics/gcp-variant-transforms
- $ cd gcp-variant-transforms/
- $ sudo apt-get install python-pip python-dev build-essential
- $ sudo pip install --upgrade pip
- $ sudo pip install --upgrade virtualenv
- $ virtualenv venv
- $ . venv/bin/activate
- $ python setup.py install

Running tests (python setup.py test) works fine at this point but running the integration test (with no command line options just to get the import error) gives me this:

$ python gcp_variant_transforms/testing/integration/run_tests.py
Traceback (most recent call last):
  File "gcp_variant_transforms/testing/integration/run_tests.py", line 25, in <module>
    from googleapiclient import discovery
ImportError: No module named googleapiclient

How do you get googleapiclient module in your setup?

Reply:
ah, sorry for the confusion... I only meant the additional dependencies related to your TODO here. We definitely need googleapiclient and bigquery api (I added these in #70), but I don't think we need others. I think you can close this PR and continue once #70 is merged (I decided to run the tests myself and ended up fixing some of the other issues as well).

Reply:
So for future reference, I created Issue #71 and updated the development guide as well. Closing this PR and ignoring the dependency conflicts for now.